<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDetailsDialog</class>
 <widget class="QDialog" name="MessageDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Block height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="m_heightLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="m_hashLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="m_amountLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Transaction hash</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Message size (bytes)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="m_sizeLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_messageTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_replyButton">
       <property name="text">
        <string>Reply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveButton">
       <property name="text">
        <string>Save to file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_prevButton">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_nextButton">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageDetailsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_replyButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageDetailsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_prevButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageDetailsDialog</receiver>
   <slot>prevClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_nextButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageDetailsDialog</receiver>
   <slot>nextClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageDetailsDialog</receiver>
   <slot>saveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>prevClicked()</slot>
  <slot>nextClicked()</slot>
  <slot>saveClicked()</slot>
 </slots>
</ui>
